client/src/app/components/stardust/history/TransactionRow.tsx

```diff
@@ -35,23 +35,28 @@ const TransactionRow: React.FC<ITransactionEntryProps> = ({
 
     const isTransactionFromStardustGenesis =
         milestoneIndex && TransactionsHelper.isTransactionFromIotaStardustGenesis(network, milestoneIndex);
-    const transactionLink = isTransactionFromStardustGenesis
-        ? `/${CHRYSALIS_MAINNET}/search/${transactionId}`
-        : `/${network}/transaction/${transactionId}`;
+    const transactionLink =
+        isTransactionFromStardustGenesis && !transactionId.includes(STARDUST_SUPPLY_INCREASE_TRANSACTION_ID)
+            ? `/${CHRYSALIS_MAINNET}/search/${transactionId}`
+            : `/${network}/transaction/${transactionId}`;
 
     return (
         <tr className={darkBackgroundRow ? "dark" : ""}>
             <td className="transaction-id">
-                <Link to={transactionLink} className="row center margin-r-t">
-                    <TruncatedId id={transactionId} />
-                    {isTransactionFromStardustGenesis && (
-                        <Tooltip tooltipContent="This link opens the transaction on Chrysalis Mainnet" childrenClass="row middle">
-                            <span className="material-icons" style={{ fontSize: "14px" }}>
-                                warning
-                            </span>
-                        </Tooltip>
-                    )}
-                </Link>
+                {isTransactionFromStardustGenesis && transactionId.includes(STARDUST_SUPPLY_INCREASE_TRANSACTION_ID) ? (
+                    <span>Stardust Genesis</span>
+                ) : (
+                    <Link to={transactionLink} className="row center margin-r-t">
+                        <TruncatedId id={transactionId} />
+                        {isTransactionFromStardustGenesis && (
+                            <Tooltip tooltipContent="This link opens the transaction on Chrysalis Mainnet" childrenClass="row middle">
+                                <span className="material-icons" style={{ fontSize: "14px" }}>
+                                    warning
+                                </span>
+                            </Tooltip>
+                        )}
+                    </Link>
+                )}
```

Comment:
This could be turned into a component to reuse in the this and the other 2 components, since they are almost identical. The only change would be giving the link a max width so that it doesn't overflow the `TransactionCard.tsx`